src/quilla/common/enums.py

```diff
@@ -38,6 +38,7 @@ class XPathValidationStates(ValidationStates):
     NOT_HAS_ATTRIBUTE = 'NotHasAttribute'
```

Comment:
Would this make it a little confusing since it is just checking that the specific attribute doesn't exist, as opposed to checking that no attribute exists?

Reply:
I did not see where it is used - if you use this status to describe a "specific" attribute instead of an entity which may have many attributes, you don't even need to have "Attribute" in the naming. "NONEXISTING" will be enough. Just my suggestion and naming really depends on the context. You can decide by yourself.

Reply:
I might just remove the NOT states, since they're only added because it was easy to do, but it is unlikely that NOT_HAS_ATTRIBUTE/PROPERTY would really come up, much in the same sense that I'm not sure "NOT_ATTRIBUTE_HAS_VALUE" is that useful